src/sql/src/catalog.rs

```diff
@@ -570,6 +569,9 @@ impl fmt::Display for CatalogError {
                 write!(f, "unknown cluster replica '{}'", name)
             }
             Self::UnknownItem(name) => write!(f, "unknown catalog item '{}'", name),
+            Self::UnexpectedType(name, cit) => {
```

Comment:
Somewhat subjective and not a big deal, but I think something like `item_type` would be better. It took me a second to figure out what `cit` meant. Feel free to ignore if you disagree.

Reply:
Fine by me!